packages/service-provider-server/src/cli-service-provider.ts

```diff
@@ -247,12 +263,18 @@ class CliServiceProvider
         connectionString.toString(),
         clientOptions,
         bus,
-        MongoClientCtor
+        MongoshMongoClient
```

Comment:
nit: Do you also want to change L257 to use this ctor? It shouldn't make a difference in behavior right now, but it seems like it would be a bit more consistent